pico_w/bt/standalone/client.c

```diff
@@ -268,6 +268,15 @@ int main() {
     // turn on!
     hci_power_control(HCI_POWER_ON);
 
+    // We're using threadsafe background where poll does nothing and work is performed in a low priority IRQ
+    // So btstack_run_loop_execute effectively does nothing and you're free to just loop and do your own stuff
+    // You would have to call btstack_run_loop_execute if you were using pico_cyw43_arch_poll
+#if 1
```

Comment:
Apologies for the stupid question, but why does this PR add `#if 1` in `client.c` but `#if 0` in `server.c` ?

Reply:
Fair enough :slightly_smiling_face:

Reply:
yeah, i also found it a bit odd(confusing); i'm planning to make some comment tweaks when i merge